src/riak_kv.proto

```diff
@@ -389,14 +389,11 @@ enum TsColumnType {
   FLOAT = 2;
```

Comment:
I prefer DOUBLE to closer match Oracle, Mysql, and postgres data type names sampled today.